changedetectionio/importer.py

```diff
@@ -182,7 +183,12 @@ def run(self,
                 days, hours = divmod(hours, 24)
                 weeks, days = divmod(days, 7)
                 extras['time_between_check'] = {'weeks': weeks, 'days': days, 'hours': hours, 'minutes': minutes, 'seconds': 0}
-
+            
+            # Openoffice is sometimes converting 'FALSE' to '=FALSE()'
+            if strtobool(str(data.get('dynamic wachet')).replace('=', '').replace('()', '').strip().lower()):
```

Comment:
this will crash the app if its not set , you cannot make `str(None)`

use ` data.get('xxxx','False')`

Reply:
and please, try to think of other people who maintain this code, this is really really hard to understand